components/layout_2020/style_ext.rs

```diff
@@ -953,6 +964,24 @@ impl ComputedValuesExt for ComputedValues {
         };
         has_percentage(box_offsets.block_start) || has_percentage(box_offsets.block_end)
     }
+
+    // <https://drafts.csswg.org/cssom-view/#overflow-directions>
+    fn scrolling_overflow_direction(&self) -> ScrollingOverflowDirection {
```

Comment:
Can we simply name the method `overflow_directions`, and the struct `OverflowDirections`, to match the spec?

Reply:
Renamed, thanks!